docker/README.md

```diff
@@ -21,25 +21,29 @@ In a directory that contains the `Dockerfile` included in this repository, on a
 
 ### Upgrade Process Recommendations
 
-The FiftyOne Teams 0.8.8 Database (version `0.16.6`) is forward-compatible with the FiftyOne Teams 0.9.2 Client (database version `0.17.2`).  Voxel51 recommends the following upgrade process:
+The FiftyOne Teams 0.10.0 Client (database version `0.18.0`) is backwards-compatible with the FiftyOne Teams 0.8.8 Database (version `0.16.6`) and all the versions between. Voxel51 recommends the following upgrade process:
```

Comment:
FiftyOne Teams 0.8.8 Client (database version `0.16.6`)

Reply:
(yes I know this logic is head-scratching either way; we'll get to simplicity...)

Reply:
the 0.10.0 Client would be connecting to a 0.8.8 Database...

Do we want to change this to reflect the fact that you can't actually use the 0.10.0 App to connect to a 0.8.8 Database?

Reply:
How about phrasing it like this?

> The FiftyOne Teams Client X.X.X is backwards-compatible with FiftyOne Teams App Y.Y.Y and all versions between

Reply:
Does the Client connect to the App?  it feels like we're talking about SDK --> Database(or even dataset) and App --> Database(or even dataset) versioning?

Reply:
This is implicitly requiring that teams-app and db are updated simultaneously.

You can take a coffee break between Step 2 and 3 in the migration guide below, but 3-6 should be run immediately